tsconfig.json

```diff
@@ -22,6 +22,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "views/homepage/OrganizationsSections"],
```

Comment:
why this?? :O

Reply:
I don't even have and idea how it got there or why, but I'll remove it now